dev-client/src/components/content/typography/TranslatedHeading.tsx

```diff
@@ -25,7 +25,7 @@ type TranslatedHeadingProps = TranslatedContentProps;
 
 export function TranslatedHeading({i18nKey, values}: TranslatedHeadingProps) {
   return (
-    <Heading variant="h1">
```

Comment:
This was an oversight in my initial implementation of the updated typography components; `h4` / `h5` are the correct variants to match the design's specified styling.